lib/dialogs.ts

```diff
@@ -23,20 +24,26 @@ export default function createDialogs(channel, ids) {
 
     // Use provided ID, default to the current extension.
     options.id = options.id || ids.extension
-
     if (options.id) {
       return channel.call('openDialog', 'extension', options)
     } else {
       throw new Error('Extension ID not provided.')
     }
   }
 
+  function openCurrentDialog(options?) {
+    if (ids.app) {
+      return openCurrentAppDialog(options)
+    } else {
+      options.id = ids.extension
```

Comment:
I'd avoid a direct object assignment here cause it changes the value in a reference.

Reply:
I think we copy it somewhere in this file before we use ( as we use assignement in the other extension method ).

If not we should fix it.

Reply:
```
return openExtensionDialog({
...options,
id: ids.extension
})
```